[#161260961] ✨(iOS) Build out Objective-C library

mcgomez opened the pull request:
## Feature ##
- Create base project with example and pod
- Create Objc version of Brytescore SDK.

## Approach ##
--

## Resources ##
--

mcgomez committed:
[#161260961] ✨(iOS) Build out Objective-C library

- Create base project with example and pod

mcgomez committed:
[#161260961] ✨(iOS) Build out Objective-C library

- Create Objc version of Brytescore SDK.

emilyemorehouse commented on BrytescoreObjCAPI/Example/BrytescoreObjcAPI/ViewController.m:
```diff
@@ -0,0 +1,242 @@
+    
+    apiManager = [[BrytescoreAPIManager alloc] initWithAPIKey:@"abc123"];
+    
+	blue = [UIColor colorWithRed:0.15 green:0.66 blue:0.88 alpha:0.8];
```
The indentation is off here (it copies out of Github as a tab instead of spaces, that may be the issue)

mcgomez replied:
I don't see any indentation issues in xcode. It shows that my indentation is set to 4 spaces

emilyemorehouse replied:
Hm. It looks like line 39 has a tab?

<img width="576" alt="screen shot 2018-10-26 at 9 30 13 am" src="https://user-images.githubusercontent.com/4264897/47569581-f4cbed00-d901-11e8-8ce8-f202dc310ca6.png">

```suggestion
    blue = [UIColor colorWithRed:0.15 green:0.66 blue:0.88 alpha:0.8];
```

mcgomez replied:
Yeah I don't know what xcode is doing hahaha

emilyemorehouse commented on BrytescoreObjCAPI/BrytescoreObjCAPI/Classes/BrytescoreAPIManager.m:
```diff
@@ -0,0 +1,787 @@
+
+#import "BrytescoreAPIManager.h"
+
+//@interface BrytescoreAPIManager()
```
We can get rid of this interface section since we have the header file

emilyemorehouse commented on BrytescoreObjCAPI/BrytescoreObjCAPI/Classes/BrytescoreAPIManager.m:
```diff
@@ -0,0 +1,787 @@
+}
+
+/**
+ Generate RFC4112 version 4 compliant UUID using Swift's built-in generator
```
Update for Obj-C :)

emilyemorehouse commented on BrytescoreObjCAPI/Example/BrytescoreObjcAPI/ViewController.m:
```diff
@@ -0,0 +1,242 @@
+//
```
Can you open up the main files that we interact with (this one and the `BrytescoreAPIMangager.m` file) in Atom and resave them? I think Atom by default will remove any trailing whitespace or will clean out lines that are all spaces.

emilyemorehouse commented on BrytescoreObjCAPI/BrytescoreObjCAPI/Classes/BrytescoreAPIManager.m:
```diff
@@ -0,0 +1,787 @@
+static NSString* packageName = @"/package.json";
+static NSString* hostname = @"com.brytecore.mobile";
+static NSString* library = @"iOS";
+static NSString* libraryVersion = @"0.0.0";
```
Should we also be updating this to match the version when we deploy? I can't remember if we do that in the other versions, but we probably need to

mcgomez replied:
Yeah, we didn't do that in the other version. But I'll make that change so next push will include it

mcgomez committed:
[#143368985] PR / Code Review

- Remove whitespace and empty/unused comments.
- Update documentation for generateUUID.
- Update library version number.

emilyemorehouse committed:
Update BrytescoreObjCAPI/Example/BrytescoreObjcAPI/ViewController.m

Co-Authored-By: mcgomez <marisa.gomez25@gmail.com>
